packages/security/__tests__/DefaultCredentialManager.test.ts

```diff
@@ -190,9 +209,53 @@ describe("DefaultCredentialManager", () => {
           await privateManager.saveCredentials(values.account, values.credentials);
 
           expect(privateManager.checkForKeytar).toHaveBeenCalledTimes(1);
+          // tslint:disable-next-line no-magic-numbers
+          expect(keytar.deletePassword).toHaveBeenCalledTimes(5);
           expect(keytar.setPassword).toHaveBeenLastCalledWith(privateManager.service, values.account, values.credentials);
         });
       });
+
+      describe("Windows credential management", () => {
+        const oldPlatform = process.platform;
+
+        beforeAll(() => {
+          Object.defineProperty(process, "platform", { get: () => "win32" });
+        });
+
+        afterAll(() => {
+          Object.defineProperty(process, "platform", oldPlatform);
+        });
+
+        it("should load credentials that exceed max length", async () => {
+          (keytar.getPassword as jest.Mock).mockReturnValueOnce(null)
+                                           .mockReturnValueOnce(longValues[0].credentials)
+                                           .mockReturnValueOnce(longValues[1].credentials);
+
+          expect(await privateManager.loadCredentials(values.account)).toEqual(values.credentials);
+          expect(keytar.getPassword).toHaveBeenLastCalledWith(service, longValues[1].account);
+        });
+
+        it("should save credentials that exceed max length", async () => {
+          // tslint:disable no-magic-numbers
+          const longCredentials = values.credentials.repeat(512);
+          const numFields = Math.ceil(longCredentials.length / 2560);
```

Comment:
should 2560 be pulled from a Constant?

Reply:
Sure, we can change `DefaultCredentialManager.WIN32_CRED_MAX_STRING_LENGTH` from private to public 🙂